schema/va-spec/base/insilico-tools.yml

```diff
@@ -0,0 +1,98 @@
+$schema: "https://json-schema.org/draft/2020-12/schema"
+$id: "https://w3id.org/ga4gh/schema/va-spec/1.x/profiles/assay-var-effect-source.yaml"
+title: VA Spec Molecular Consequence statement result Profiles
+strict: true
+
+imports:
+  va-core: ./va-core-source.yaml
+  cat-vrs: ../../cat-vrs/cat-vrs-source.yaml
+  vrs: ../../vrs/vrs-source.yaml
+  gks-core: ../../gks-core/gks-core-source.yaml
+
+$defs:
+  InSilicoVariantEffectScoreStudyResult:
+      inherits: va-core:StudyResult
+      maturity: draft
+      type: object
+      description: >-
+        A StudyResult that reports a variant effect score from a functional assay.
+        Notes: Larry and Alex to discuss specifiedBy/sourceDataset/reportedIn to determine where everything will go
+      properties:
+        type:
+          extends: type
+          const: "InSilicoVariantEffectScoreStudyResult"
+          default: "InSilicoVariantEffectScoreStudyResult"
+          description: MUST be "InSilicoVariantEffectScoreStudyResult".
+        label:
+          type: string
+          description: A primary name for the score
+        focusVariant:
+          extends: focus
+          oneOf: 
+            - $refCurie: gks.core:iriReference
+            - $refCurie: vrs:Allele
+          description: The variant for which this annotation applies to.
+        transcripts:
+          type: array
+          ordered: false
+          items:
+            oneOf:
+              - $refCurie: gks.core:iriReference
+              - $refCurie: vrs:SequenceReference
+          description: Transcripts for which this score is applicable
+        score:
+          type: number
+          description: The score of the variant effect.
+        ancillaryResults:
+          type: object
+          description: Dictionary of any normalized or ancillary scores (e.g., the CADD Phred score)
+        sourceDataSet:
+          description: >-
+            The full data set that this measurement is a part of
+          extends: sourceDataSet
+
+  InSilicoVariantEffectPredictionEvidenceLine:
+      inherits: va-core:EvidenceLine
+      maturity: draft
+      type: object
+      description: >-
+        An EvidenceLine that reports a variant effect prediction from a functional assay.
+      properties:
+        type:
+          extends: type
+          const: "InSilicoVariantEffectPredictionEvidenceLine"
+          default: "InSilicoVariantEffectPredictionEvidenceLine"
+          description: MUST be "InSilicoVariantEffectPredictionEvidenceLine".
+        targetProposition:
+          $ref: "#/$defs/Proposition"
+          description: >-
+            The possible fact against which evidence items contained in an Evidence Line were collectively evaluated. This
+            will likely be a statement of pathogenicity or oncogenicity.
+        hasEvidenceItems:
+          type: array
```

Comment:
This could be confusing - I would expect EvidenceItems to contain the evidence is used to calculate the predictive score, rather than the score itself. 
For example, CADD uses over 100 possible pieces of evidence including conservation scores, overlap of protein domain, pathogenicity predictions, splice impact predictions, overlap with regulatory regions, number of common variant nearby.